lib/Mojo/WebSocketProxy/Backend/ConsumerGroups.pm

```diff
@@ -73,6 +76,27 @@ sub loop {
     };
 }
 
+sub category_config {
+    my $self = shift;
+    return $self->{category_config} //= do {
+        my %processed = ();
+        try {
+            my $raw_config = YAML::XS::LoadFile($self->category_map_config);
```

Comment:
I'd try to keep this logic inside `binary-websocket-api`. and pass this configuration when we create backend. IMHO reading configuration is out of scope for this package